tests/conftest.py

```diff
@@ -16,13 +17,15 @@
 @pytest.fixture()
 def app(request):
     """Return Flask application for the pytest session."""
-    return _make_app(request, "cachito.web.config.TestingConfig")
+    yield _make_app(request, "cachito.web.config.TestingConfig")
```

Comment:
I remember you explaining a little bit of the context/backstory behind this change to me on Friday... Can you refresh my memory? :pray:

Reply:
Updated, and commented at the yields as well.